tempesta_fw/t/functional/helpers/deproxy.py

```diff
@@ -382,12 +430,12 @@ def update(self):
 
     @staticmethod
     def create(status, headers, version='HTTP/1.1', date=False,
-               srv_version=None, body=None):
+               srv_version=None, body=None, body_void=False):
```

Comment:
This way we'll skip sanity check in `helpers.HttpMessage.read_sized_body()` if an empty body is incorrectly passed together with a non-zero `Content-Length: ` HTTP header. I'd leave this an explicit parameter.